### ISSUE TITLE
[Bug] Preview is not taking full height of the editor

### ISSUE THREAD

**Issue by yannbf** [storybookjs/vs-code-plugin#22]
Thank you so much for developing this addon! I see so much potential in it! Here's a bug that is getting on my way currently:

### Describe the bug

When loading Aesop, I'd expect my Storybook to take the full height of the editor, but unfortunately that's not the case.

### Steps to reproduce the behavior

1. Awaken aesop
2. See the gap at the bottom

### Expected behavior

The preview takes full height so users can better visualize the addon panel without feeling cluttered.

### Screenshots and/or logs

![image](https://user-images.githubusercontent.com/1671563/107996897-d06c1100-6fe1-11eb-80f8-5d10d9c2df10.png)

### Environment

 - OS: macOS Catalina
 
 ### Extra remarks
 
 I also have the feeling that the webview is scaled up. The fonts are bigger, it feels too tight.

**Comment by giacomo9999** [comment]
Yeah, that is weird...I'll have a look. There are some...unusual disparities between the way VSCode renders pages and how Chrome does it.
